tests/github-retag-and-push.bats

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bats
+
+# Place the following before an assertion to the the output
+# echo "output = ${output}"
+
+docker pull busybox > /dev/null
+
+teardown () {
+    unset GITHUB_SHA
+    unset GITHUB_REF
+}
+
+
+@test "valid other branch" {
+    export GITHUB_REF=refs/heads/foobar
+    export GITHUB_SHA=12345678
+    run docker tag busybox libero/otherbranch:12345678
+    run github/retag-and-push.sh otherbranch 12345678
+    [ "$status" -eq 0 ]
+    run docker pull ${DOCKER_REGISTRY}liberoadmin/otherbranch:foobar-12345678
+    [ "$status" -eq 0 ]
+    run docker pull ${DOCKER_REGISTRY}liberoadmin/otherbranch:latest
+    [ "$status" -eq 1 ]
+}
+
+@test "valid master tagging" {
+    export GITHUB_REF=refs/heads/master
+    export GITHUB_SHA=12345678
+    run docker tag busybox libero/my-dummy-project:12345678
+    run github/retag-and-push.sh my-dummy-project 12345678
+    [ "$status" -eq 0 ]
+    run docker pull ${DOCKER_REGISTRY}liberoadmin/my-dummy-project:master-12345678
+    [ "$status" -eq 0 ]
+    run docker pull ${DOCKER_REGISTRY}liberoadmin/my-dummy-project:latest
+    [ "$status" -eq 0 ]
+}
+
+@test "rename liberoadmin to libero" {
+    export GITHUB_REF=refs/heads/master
+    export GITHUB_SHA=12345678
+    run docker tag busybox liberoadmin/my-dummy-project:12345678
+    run github/retag-and-push.sh my-dummy-project 12345678
+    [ "$status" -eq 0 ]
+    run docker pull ${DOCKER_REGISTRY}liberoadmin/my-dummy-project:master-12345678
+    [ "$status" -eq 0 ]
+    run docker pull ${DOCKER_REGISTRY}liberoadmin/my-dummy-project:latest
+    [ "$status" -eq 0 ]
+}
+
+@test "valid semver tagging" {
+    export GITHUB_REF=refs/tags/v1.2.43
+    export GITHUB_SHA=12345678
+    run docker tag busybox libero/my-dummy-project:master-12345678
+    run github/retag-and-push.sh my-dummy-project 12345678
+    [ "$status" -eq 0 ]
+    run docker pull ${DOCKER_REGISTRY}liberoadmin/my-dummy-project:1
+    [ "$status" -eq 0 ]
+    run docker pull ${DOCKER_REGISTRY}liberoadmin/my-dummy-project:1.2
+    [ "$status" -eq 0 ]
+    run docker pull ${DOCKER_REGISTRY}liberoadmin/my-dummy-project:1.2.43
+    [ "$status" -eq 0 ]
+}
+
+@test "missing v in tag" {
```

Comment:
did we specify the `v` requirement somewhere? No preference on the choice personally, but [the only existing tags](https://github.com/libero/event-bus) I can find don't use it.

Reply:
I normally use it (https://github.com/libero/php-coding-standard/releases).

Reply:
The ADR uses `v` in the example, but does not specify its use.  In my opinion this is an oversight.  As the script relies on `GITHUB_REF` we need to agree on a convention regarding this.

Reply:
Proposing to edit the ADR (but only because it's in a `Proposed` state still) to mandate the `v`.

Reply:
Opened a WIP PR for this https://github.com/libero/community/pull/38/files